### PR TITLE
facial expression recognition demo update (FPS added) 

### PR DESCRIPTION
**Pull request by negarhdr**
This PR adds FPS to the webcam demo of facial expression recognition.

### CODE REVIEW

**Review by passalis — DISMISSED** [review]
Thank you!

**Review by tsampazk — CHANGES_REQUESTED** [review]
Thanks! Just a minor thing.

**Comment by negarhdr** [comment]
> Thanks! Just a minor thing.

Thanks for checking.

**Review by tsampazk — DISMISSED** [review]
Thank you :smile:

File: projects/python/perception/facial_expression_recognition/image_based_facial_emotion_estimation/inference_demo.py [hunk 1]
```diff
@@ -246,7 +255,8 @@ def main():
     learner = FacialEmotionLearner(device=args.device, ensemble_size=args.ensemble_size, dimensional_finetune=False,
                                    categorical_train=False)
     learner.init_model(num_branches=args.ensemble_size)
-    model_path = learner.download(mode="pretrained")
+    # model_path = learner.download(mode="pretrained")
+    model_path = "./esr_9"
```
Comment (tsampazk) [inline projects/python/perception/facial_expression_recognition/image_based_facial_emotion_estimation/inference_demo.py:259]: This change needs to be reverted probably?

**Review by ad-daniel — APPROVED** [review]
Thank you!

**Review by tsampazk — COMMENTED** [review]
Applied some PEP8 fixes that failed the tests. Just noticed, is this indeed meant to merge into master?

**Review by ad-daniel — APPROVED** [review]
It's fine since it's a demo and doesn't impact functionality, thank you!

**Review by tsampazk — APPROVED** [review]

**Pull request merged**